zen/providers/offline.py

```
@@ -129,107 +131,94 @@ class OllamaProvider:
-            )
+            result = subprocess.run(["which", "ollama"], capture_output=True, text=True, timeout=5)
             if result.returncode != 0:
                 return False
-            
+
             # Check if server is running
             import requests
+
             response = requests.get(f"{self.base_url}/api/tags", timeout=2)
             return response.status_code == 200
         except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)